tests/unit_tests/test_obstacle_map.py

```diff
@@ -1,9 +1,9 @@
 from SDA import *
 import unittest
-import numpy 
+import numpy
 
 class Test_Obstacle_map(unittest.TestCase):
-	
+
 	def setUp(self):
```

Comment:
[setUp() function name should be all lower case and using underscores instead of spaces in accordance w/ pep8 standards.](https://www.python.org/dev/peps/pep-0008/#function-names)

Reply:
@DarkmatterVale, should this function be called `set_up()` or `__init__()`?

Reply:
This is special for the unit test library and must stay the same.

Reply:
Alright—got it.